components/evaluators/evaluators-list.tsx

```diff
@@ -1,34 +1,39 @@
+import { use } from "react";
 import { ATTESTORS_PER_PAGE } from "../../configs/ui";
 import EvaluatorsListRow from "./evaluators-list-row";
-import EvaluatorsListSkeleton from "./evaluators-list-skeleton";
 import EvaluatorsPagination from "./evaluators-pagination";
-import { Suspense } from "react";
 import { getTrustedAttestors } from "../../github/getTrustedAttestors";
 
-async function EvaluatorsListInner({
+// Implement caching and revalidation
+async function getAttestorsData() {
+  const attestors = await getTrustedAttestors();
+  return attestors;
+}
+
+export default function EvaluatorsList({
   searchParams,
 }: {
   searchParams: Record<string, string>;
 }) {
-  const attestors = await getTrustedAttestors();
   const currentPage = Number(searchParams?.p) || 1;
+  const attestors = use(getAttestorsData());
```

Comment:
`use` seems to be a canary/preview only function, FYI. I'm surprised it's available as we don't seem to be on either of those versions of react.

https://react.dev/reference/react/use

Reply:
Probably NextJS team already implemented exposes it: https://nextjs.org/blog/next-14-2